### PR TITLE
Fix broken form of keyboard shortcut modal page

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Fix broken form of keyboard shortcut help modal

### Screenshots (if appropriate)
**Before**
screen shot 2016-02-15 at 1 05 49 pm

**After**
screen shot 2016-02-15 at 1 05 40 pm

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by corneadoug** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
LGTM

**Comment by AhyoungRyu** [comment]
Looks Good!

**Comment by anthonycorbacho** [comment]
+1

File: zeppelin-web/src/components/modal-shortcut/modal-shortcut.html [hunk 1]
```diff
@@ -125,7 +125,6 @@ <h4 class="modal-title" id="myModalLabel">Keyboard shortcuts</h4>
         <div class="row">
           <div class="col-md-4">
             <div class="keys">
-              <kbd class="kbd-dark">Ctrl</kbd> + <kbd class="kbd-dark">{{ isMac ? 'Option' : 'Alt'}}</kbd> + <kbd class="kbd-dark">o</kbd>
               <kbd class="kbd-dark">Ctrl</kbd> + <kbd class="kbd-dark">Alt</kbd> + <kbd class="kbd-dark">r</kbd>
```
Comment (felixcheung) [inline zeppelin-web/src/components/modal-shortcut/modal-shortcut.html]: it looks like this `Alt` should be `{{ isMac ? 'Option' : 'Alt'}}`? Basically everywhere we have Alt should say Option on Mac

Comment (minahlee) [inline zeppelin-web/src/components/modal-shortcut/modal-shortcut.html]: I completely missed that! Thank you for the review as always.

**Comment by felixcheung** [comment]
LGTM!!

**Comment by prabhjyotsingh** [comment]
LGTM

**Pull request closed without merging**